R/util-func-namespace.R

```diff
@@ -0,0 +1,15 @@
+##' getFunctionIfNamespaced looks to see if a strFunction is namespaced and looks it up
+##' allowing the do.call in run step to process correctly
+##' this will either return a function, or the original string if not namespaced
```

Comment:
Can we export this? Once we have it nailed down, I'll use it in gsm.app for plugin functions. I'd genericize the name a little, too, since this function will parse strFunction whether or not it's namespaced.